.github/workflows/build-internal.yml

```diff
@@ -29,6 +29,17 @@ jobs:
           eas-version: latest
           token: ${{ secrets.EXPO_TOKEN }}
 
+      - name: Create .env file from secrets
+        run: |
+          echo "EXPO_PUBLIC_CONTACT_ADDRESS=${{ secrets.EXPO_PUBLIC_CONTACT_ADDRESS }}" >> .env
```

Comment:
the way I set it up right now, it needs the file to be called `.env.production` for prod, and `.env` for preview. That makes it easier to have both locally, but if you want to change that we can

Reply:
That's fine, I'll make the update here